src/main/java/com/jadventure/game/entities/Player.java

```diff
@@ -574,6 +567,18 @@ public void checkForCharacterChange() {
         }
     }
 
+    private String getString(int highestCharacterLevel, String highestCharacter, Iterator it) {
+        while (it.hasNext()) {
+            Entry pairs = (Entry)it.next();
+            int value = (int)pairs.getValue();
+            if (value > highestCharacterLevel) {
+                highestCharacterLevel = value;
+                highestCharacter = (String)pairs.getKey();
+            }
+        }
+        return highestCharacter;
+    }
+
```

Comment:
This method essentially functions to find the name of highest "character" level. I suggest two changes:

- The name is super vague, I would change it to something like `getHighestCharacter`
- Since all of the parameters are based on data from this class, I would instantiate them inside this method rather than pass arguments